### PR TITLE
Created newsLetter, bug fixes in cardBox

### PR DESCRIPTION
**Pull request by Sarvesh025**
newsLetter

### CODE REVIEW

File: src/component/cardBox/index.tsx [hunk 1]
```diff
@@ -1,84 +1,144 @@
...
     }
-
-    useEffect(() => {
-
-        window.addEventListener("resize", handleResize, false);
+        
+    
+    useEffect(() => {        
+            
         const isMobile = () => {
```
Comment (arijeet-mukherjee) [inline src/component/cardBox/index.tsx:66]: we can remove this method, already util having same code for isMobile check, reuse that

File: src/data.json [hunk 2]
```diff
@@ -339,6 +342,20 @@
...
+        "image_position": "right",
+        "button_name": "Subscribe",
+        "button_icon": "/arrowrightwhite.svg",
+        "paddingLeftContent": "calc((100vw / 1920) * 115)",
+        "inputBox": "Email*",
+        "bulletPointImg": "/tickmark.svg",
+        "bulletPoints":[
+            "Stay up-to-date with the trends, so you can focus your security effort", "Increase your security posture"
+        ],
+        "goTo": "/google.com"
```
Comment (arijeet-mukherjee) [inline src/data.json:358]: give it # for now, dont use google.com

File: src/app/page.tsx
Comment (arijeet-mukherjee) [inline src/app/page.tsx:1]: Do verify no console log errors are coming

File: src/data.json (hunk 2 above)
Comment (Sarvesh025) [reply src/data.json]: Resolved in the new commit

File: src/app/page.tsx
Comment (Sarvesh025) [reply src/app/page.tsx:1]: Verified, No errors are there.

**Review by arijeet-mukherjee — APPROVED** [review]
LGTM

**Pull request merged**